fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/KeyValueCursorTest.java

```diff
@@ -44,6 +44,8 @@
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
```

Comment:
It would probably be good to add a test here where there is a scan over a single key range. The expectation would be that the continuation from the (only) key should be empty in the old serialization mode, but it should be non-empty in the new serialization mode

Reply:
I'm not sure I see the test over a single key range, but maybe I'm missing it?